lib/index.ts

```diff
@@ -35,6 +35,9 @@ function parsePCBDesign(sexprRoot: any[]): any {
 
   const [pcbLiteral, filePath, ...sexprMainContent] = sexprRoot
 
+  result.pcb = pcbLiteral
+  result.file = filePath
```

Comment:
This is kind of my fault but if we're manipulating result like this we should give it a type (in the lines above this)

```ts
const result: z.infer<PCBDesignSchema> = {} as any
```

It'll definitely help us correctly put together the file